### DISCUSSION TITLE
Roidmi vacuum Eva v66

### DISCUSSION THREAD

**Discussion by nish1982** [merdok/homebridge-miot#305] [Ideas]
Hi,  is it possible to add the V66 Roidmi Eva robot cleaner?

**Comment by merdok** [comment]
> Hi,  is it possible to add the V66 Roidmi Eva robot cleaner?

It certainly would be, but I would need the unique model id from that device so I can look up the spec.

**Reply by nish1982** [reply]
sorry for the delay

https://merdok.github.io/miotspec/?model=roidmi.vacuum.v66

Model is roidmi.vaccuum.v66

it’s new Eva model

**Reply by merdok** [reply]
Thanks, full support will be added in the next update. You can already add it now and the plugin will try to do some stuff automatically as much as it can so it might already work.

**Comment by nish1982** [comment]
Unfortunately it does pick up a number of actions but just timeouts when doing anything.

**Reply by merdok** [reply]
Timeouts usually mean the token might be wrong or it requires a micloud connection...
